src/client/canvas/renderer.ts

```diff
@@ -13,7 +13,7 @@ import { PolygonGeometry } from './geometry/polygon_geometry'
 import { TextGeometry } from './geometry/text_geometry'
 import { Group } from './object/group'
 import { Object2d } from './object/object2d'
-import { Shape } from './object/shape'
+import { Geometry , Shape } from './object/shape'
 
 export class CanvasRenderer {
```

Comment:
Can we rename this file to `canvas_renderer`?

Reply:
I had intended to either rename this or delete it, depending on how we felt about svg rendering vs canvas rendering

Reply:
I don't think we're ready for deletion yet, especially since I'm not sure how it will perform vs SVG for more complex scenes. Chart will be an interesting test. I say we rename it for now.